packages/carbon-addons-boomerang-react/src/components/FeatureHeader/_featureHeader.scss

```diff
@@ -3,13 +3,50 @@
     background-color: $ui-01;
     display: flex;
     justify-content: space-between;
+    min-height: 6.25rem;
+    padding: 1.5rem 1.5rem 0;
     position: sticky;
     top: 0;
     z-index: 999;
+    &.--bordered {
+      border-bottom: 0.0625rem solid #c1c7cd;
+    }
   }
+
+  .#{$prefix}--bmrg-feature-header__nav {
+    margin-bottom: auto;
+  }
+
   .#{$prefix}--bmrg-feature-header__content {
     display: flex;
     flex-direction: column;
-    justify-content: space-between;
+    justify-content: center;
```

Comment:
Not sure what is best to do here. The nav and footer are aligned to the top and bottom so thought it made sense to center the children and header (title & subtitle) text